src/Native/System.Net.Security.Native/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+project(System.Net.Security.Native)
+
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+add_definitions(-DPIC=1)
+
+if (APPLE)
```

Comment:
Sorry we were not aware of this. We were checking for precedents and found one at https://github.com/dotnet/corefx/blob/2c8ce526bf9fbb466181397269a3b74b728d6978/src/Native/CMakeLists.txt#L79 @sokket, would you be open to tracking this as a separate issue?